TheAltF4Stream/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.12.1
+RUN apk add --no-cache \
+  build-base \
+  cmake \
+  automake \
+  autoconf \
+  libtool \
+  pkgconf \
+  coreutils \
+  curl \
+  unzip \
+  gettext-tiny-dev \
+  git
+RUN git clone --branch "nightly" --progress "https://github.com/neovim/neovim.git" \
```

Comment:
You should use '--dept=1' for the git clone to speed up the clone time. You don't care about the whole history.

Reply:
Appreciate it ! Will update once I get back!